lib/WUI/sntp/sntp_client.c

```diff
@@ -2,8 +2,8 @@
 #include "sntp_client.h"
 #include "wui_api.h"
 
-static ip_addr_t ntp_server; // testing ntp server located in Prague
-
+static ip_addr_t ntp_server;         // testing ntp server located in Prague
+static bool sntp_processing = false; // describes if sntp is currently running or not
```

Comment:
probably `sntp_running` will be a better name

Reply:
commit `e278f27`